app/views/admin/users/widgets/_credits.html.erb

```diff
@@ -23,20 +23,22 @@
       <div class="flex gap-2 items-center">
         <select name="" id="balance-type" class="crayons-select w-auto" required>
           <option value="add">Add</option>
-          <option value="subtract">Subtract</option>
+          <% if current_credits.positive? %>
+            <option value="subtract">Subtract</option>
+          <% end %>
         </select>
 
-        <input type="number" inputmode="numeric" placeholder="10" size="5" min="1" max="9999" required class="crayons-textfield w-auto">
+        <input type="number" inputmode="numeric" size="5" min="1" max="9999" value="1" required class="crayons-textfield w-auto">
```

Comment:
The placeholder has colour contrast issues and having a default value of 1 seemed to make sense as well as indicate it's a number field.